### PR TITLE
Update tests for commands.

### PR DESCRIPTION
**Pull request by bharathcs**

### CODE REVIEW

File: src/main/java/donnafin/logic/commands/RemoveCommand.java [hunk 1]
```diff
@@ -27,7 +27,9 @@
             + ": removes a row from the attribute table. \n"
             + "Parameters: INDEX (must be a positive integer) "
             + "Example: remove 3";
-    private static final String MESSAGE_SUCCESS = "Successfully removed row.";
+    public static final String MESSAGE_SUCCESS = "Successfully removed row.";
+    public static final String MESSAGE_INVALID_TAB = "Successfully removed row.";
```
Comment (sheehui) [inline src/main/java/donnafin/logic/commands/RemoveCommand.java:31]: is this duplicate intentional?

**Review by sheehui — COMMENTED** [review]
very nice just one thing

File: src/main/java/donnafin/logic/commands/RemoveCommand.java (hunk 1 above)
Comment (bharathcs) [reply src/main/java/donnafin/logic/commands/RemoveCommand.java:31]: jeez absolutely cancerous. thanks!
```suggestion
    public static final String MESSAGE_INVALID_TAB = "Wrong tab for RemoveCommand.";
```

**Review by sheehui — APPROVED** [review]
alls good

**Pull request merged**